docs/changes/io.votable/11982.bugfix.rst

```diff
@@ -0,0 +1 @@
+Now accepting UCDs containing phot.color (Fixes: #11981).
```

Comment:
I am not sure if introducing "Fixes" here will open a can of worms or not. No other change log entries use this idiom. What do you think, @bsipocz or @saimn ?

```suggestion
Now accepting UCDs containing phot.color.
```

Reply:
Yes, I agree that it's better to cross reference issues in the pr only and leave it out from the changelog.